include/drivers/bluetooth/hci_driver.h

```diff
@@ -95,6 +96,13 @@ int bt_recv(struct net_buf *buf);
  */
 int bt_recv_prio(struct net_buf *buf);
 
+/** @brief Read static addresses from the controller.
+ *
+ * @note A weak version of this function is included in the HCI driver, so
+ *       defining it is optional per board.
+ */
+u8_t bt_read_static_addr(struct bt_hci_vs_static_addr *addrs);
```

Comment:
Since you're putting this in a public header you should document the input parameters and return value. I'm also slightly uncomfortable having such a public API which doesn't take a second parameter indicating how much space (how many entries) there is in the `addrs` array.  Btw, I think it might be nicer if we use `addrs[]` instead of `*addrs` whenever referring to an array rather than a pointer to a single entry.

Reply:
@jhedberg I agree. Is there perhaps a better header file to put this definition? I'm not certain I picked the best place.

Reply:
I can't think of anything more appropriate, so let's just go with `hci_driver.h` for now